### DISCUSSION TITLE
Autenticação: que tal possibilitarmos reconhecimento facial ou leitura de impressão digital?

### DISCUSSION THREAD

**Discussion by C6u6** [diego3g/rsxp-2023#29] [Ideas]
Sei que essa feature seria apenas um mero detalhe, mas ainda sim acho que não demandaria um tempo muito grande para ser implementada e traria uma experiência legal para o usuário.

**Comment by diego3g** [comment]
Olha, acho um incremento legal, mas como o app não tem qualquer tipo de informação sensível acho que é uma etapa a mais que talvez seja desnecessária.

Acho que poderíamos usar isso somente em coisas tipo: permitir que a pessoa desvincule um ingresso validando o Face ID ou Touch ID, ou seja, certas funções da aplicação terem essa camada de segurança a mais.

**Reply by C6u6** [reply]
Sob essa perspectiva, concordo contigo. Talvez o melhor seja mesmo implementar isso como uma camada a mais de proteção.

**Reply by andreghisleni** [reply]
ou até como uma proteçao caso alguem queira sacanear e até deslogar o app da conta do usuario, o que acham??
